distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json/Extensions.enso

```diff
@@ -135,3 +135,10 @@ Locale.to_js_object self =
     b.append ["country", self.country]
     b.append ["variant", self.variant]
     JS_Object.from_pairs b.to_vector
+
+## Converts the given value to a JSON serializable object.
+   For Map, this serialized as a Vector of Key-Value pairs.
```

Comment:
Some verb is missing here I think?

```suggestion
   For Map, this is serialized as a Vector of Key-Value pairs.
```

Reply:
Ah nvm, I remember you were mentioning that in JS keys must have String type (right?) and we allow arbitrary key types so it would not be compatible.

I think this is a place to add a short doc note about this - as otherwise further into the future someone (e.g. I 🙂) will forget what was the reason and will again be wondering why a `Map` is not mapped to the JS map-equivalent.

I'd suggest this or sth similar:

```suggestion
   For Map, this is serialized as a Vector of Key-Value pairs.
   
   Enso Maps support arbitrary types as map keys, so we cannot serialize them into JS Objects because there only strings are accepted as keys.
```